Doc/howto/functional.rst

```diff
@@ -1,34 +1,24 @@
 ********************************
-  Functional Programming HOWTO
+Functional programming in Python
```

Comment:
Perhaps capital 'P' as 'Functional Programming' is a term? Feel free to reject though, off-hand thought.

```suggestion
Functional Programming in Python
```